dist/jquery.flot.js

```diff
@@ -2015,9 +2014,12 @@ Licensed under the MIT license.
 
         function setRange(axis) {
             var opts = axis.options,
-                min,
-                max,
-                delta;
+                windowWidth = opts.windowSize,
+                min = opts.min,
+                max = opts.max,
+                delta,
+                navigateMin = opts.navigateMin || 0,
```

Comment:
in general looks good, I'd like a change of names from:
_opts.navigateMin_ to _opts.offsetBellow_
_opts.navigateMax_ to _opts.offsetAbove_